src/Esprima/JavascriptParser.cs

```diff
@@ -1329,7 +1349,10 @@ private Node ReinterpretExpressionAsPattern(Node expr)
                 {
                     if (property is Property p)
                     {
-                        p._value = ReinterpretExpressionAsPattern(p.Value);
+                        if (p._value is not null)
+                        {
+                            p._value = ReinterpretExpressionAsPattern(p.Value);
+                        }
```

Comment:
I'm puzzled about this change. In what circumstances can a property value be `null` in binding patterns? Am I missing here something? (If so, that would have consequences for the AST model and other features like AST to JS... )

Reply:
I probably need to check the (now enabled) test suite and check if the breakpoint is hit and in which conditions.